flask_monitoringdashboard/database/__init__.py

```diff
@@ -27,115 +27,168 @@
 
 
 class Endpoint(Base):
-    """ Table for storing information about the endpoints. """
+    """Table for storing information about the endpoints."""
 
     __tablename__ = '{}Endpoint'.format(config.table_prefix)
+
     id = Column(Integer, primary_key=True)
-    # name of the endpoint
+
     name = Column(String(250), unique=True, nullable=False)
-    # 0 - disabled, 1 - performance, 2 - outliers, 3 - profiler + outliers
+    """Name of the endpoint."""
+
     monitor_level = Column(Integer, default=config.monitor_level)
-    # the time when the endpoint was added
+    """0 - disabled, 1 - performance, 2 - outliers, 3 - profiler + outliers"""
+
     time_added = Column(DateTime, default=datetime.datetime.utcnow)
-    # the version when the endpoint was added
+    """Time when the endpoint was added."""
+
     version_added = Column(String(100), default=config.version)
-    # the time the endpoint was last used
+    """Version when the endpoint was added into the DB."""
+
     last_requested = Column(DateTime)
+    """Time when the endpoint was last requested."""
 
 
 class Request(Base):
-    """ Table for storing measurements of requests. """
+    """Table for storing measurements of requests."""
 
     __tablename__ = '{}Request'.format(config.table_prefix)
+
     id = Column(Integer, primary_key=True)
+
     endpoint_id = Column(Integer, ForeignKey(Endpoint.id))
-    # the processing time of the request
+    endpoint = relationship(Endpoint, backref='requests')
+    """The endpoint that handles the request."""
+
     duration = Column(Float, nullable=False)
-    # the time when the request was handled
+    """Processing time of the request in milliseconds."""
+
     time_requested = Column(DateTime, default=datetime.datetime.utcnow)
-    # the version when the request was handled
+    """Moment when the request was handled."""
+
     version_requested = Column(String(100), default=config.version)
-    # a criteria by which the requests can be grouped
+    """Version when the request was handled."""
+
     group_by = Column(String(100), default=None)
-    # the ip address of the requester
+    """Criteria which can be used for grouping multiple requests."""
+
     ip = Column(String(100), nullable=False)
-    # the HTTP status code of the request
+    """IP address of the requester."""
+
     status_code = Column(Integer, nullable=True)
+    """HTTP status code of the request."""
 
-    endpoint = relationship(Endpoint)
-    stack_lines = relationship('StackLine', back_populates='request')
-    outlier = relationship('Outlier', uselist=False, back_populates='request')
+    outlier = relationship("Outlier", uselist=False, back_populates='request')
```

Comment:
The StackLine table looks like this:
```
class StackLine(Base):
    ...
    request = relationship(Request, backref="stack_lines")
```

Since it is using `backref` (instead of `back_populates`), it adds the property `stack_lines` to Request.

Reply:
You're right. I didn't see you updated the `request` relationship in `StackLine`